src/components/SubmitSearchControl/index.css

```diff
@@ -0,0 +1,104 @@
+/** @define SubmitSearchControl; */
+
+.SubmitSearchControl-commands {
+  display: flex;
+}
+
+.SubmitSearchControl-command {
+  appearance: none;
+  background: #e9e9e9 url("./glass.svg") no-repeat center center;
+  border-radius: 0;
+  box-shadow: 0.125rem 0.125rem #a5a5a5;
+  color: #ffffff;
```

Comment:
In other places we shorted 6 identical values to 3.

``` diff
- color: #ffffff;
+ color: #fff;
```

Reply:
It looks like it's inconsistent throughout. I'll create a separate issue for it.